### ISSUE TITLE
Add HTML5 goodies

### ISSUE THREAD

**Issue by JivamuktiWeb** [Jivamukti/drupal7-compose#1]
Need to add Drupal contrib modules: 

elements
fences
telephone

**Comment by ForestMars** [comment]
I just assigned this ticket to myself.

**Comment by ForestMars** [comment]
Added these modules to composer.json and ran build/update.sh, but didn't include this ticket number + keyword to automatically close this ticket.

**Comment by JivamuktiWeb** [comment]
Added extra commit message which you can see automatically closes this ticket, once it is merged into the default branch (master).

**Comment by thinkyhead** [comment]
test
